ReChord_frontend.py

```diff
@@ -92,14 +95,27 @@ def search_snippet(path, snippet):
     Return: rendered result page 'ReChord_result.html'
     """
     xml = BytesIO(snippet.encode())
```

Comment:
It's better to explicitly declare `error_msg` to be `None`, instead of leaving it undefined if there are no errors.